src/interface/src/app/plan/scenario-details/scenario-details.component.html

```diff
@@ -1,11 +1,15 @@
-<div class="scenario-details-container">
-  <div class="details-panel">
+<div class="scenario-details-panel mat-elevation-z2"
```

Comment:
As a future improvement, perhaps we could use [content projection](https://angular.io/guide/content-projection) to make the entire expand panel a reusable component, instead of just reusing the animations.

Reply:
Good idea!!